pages/clustering/high-availability.mdx

```diff
@@ -266,22 +268,14 @@ This operation will result in writing to the Raft log.
 
 You can check the state of the whole cluster using the `SHOW INSTANCES` query. The query will show which instances are visible in the cluster,
 which network ports they are using for managing cluster state, whether they are considered alive from the coordinator's
-perspective, their role (are they MAIN, REPLICA, LEADER, FOLLOWER or unknown if not alive) and the time elapsed in ms since the last successful response
-from instances to the leader's ping. This query can be run both on leader and followers with the sole difference that followers cannot know the 
-health status of other instances and the time of last response since only the leader is pinging instances for their status.
+perspective, their role (are they MAIN, REPLICA, LEADER, FOLLOWER or unknown if not alive) and the response time
+from instances to the leader's ping. This query can be run both on the leader and followers. Followers will try contacting the leader to get the state of the cluster. 
+If contacting the leader fails, followers will return time since the last successful response and health state set to 0 and unknown, respectfully.
```

Comment:
I think you mean "respectively". However, I'm still having trouble understanding what happens when contacting the leader fails. Perhaps make an ordered list to clearly separate the results?

Reply:
Can you explain which part is problematic, I have trouble fixing it if I am not sure what the exact issue is with understanding :smile: 
I will rewrite what is happening, but I think a better way on commenting a sentence would be to say which part exactly is not understood

Reply:
Note: the [development PR](https://github.com/memgraph/memgraph/pull/2138) is marked as `breaking`. Due to that, if all comments are not resolved until the moment we release 2.18.1, I'll create new issues to resolve the remainder of the open issues after the release.